packages/component-library/draft/Kaizen/Tabs/styles.scss

```diff
@@ -1,52 +1,75 @@
 @import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/component-library/styles/border";
 @import "~@kaizen/component-library/styles/color";
 @import "~@kaizen/component-library/styles/layout";
 @import "~@kaizen/component-library/styles/type";
 
 .container {
   @include ca-margin($bottom: $ca-grid);
-  border-bottom: 1px solid $ca-border-color;
+  border-bottom: 1px solid add-alpha($kz-color-wisteria-700, 10%);
   display: flex;
 }
 
 .tab {
-  $underline-height: 4px;
-
-  @include ca-padding($top: $ca-grid * 0.5, $bottom: $ca-grid * 0.75);
```

Comment:
No need for top/bottom padding as the text is now centered and the component has a fixed height